test/cqlpy/test_materialized_view.py

```diff
@@ -1547,3 +1548,10 @@ def test_view_in_system_tables(cql, test_keyspace):
             assert view in res
             res = [ f'{r.table_name}.{r.index_name}' for r in cql.execute('select * from system."IndexInfo"')]
             assert view not in res
+
+# Test view representation in API
```

Comment:
Nitpick (not important now): the word "API" is too vague - we have a bazillion APIs. I would use the phrase "REST API". We even call the object rest_api you used in the test.

Reply:
Did you see this commentr?